cmd/pd-server/main.go

```diff
@@ -295,6 +299,6 @@ func start(cmd *cobra.Command, args []string, services ...string) {
 }
 
 func exit(code int) {
-	log.Sync()
+	log.Sync() // nolint:errcheck
```

Comment:
```suggestion
	_ = log.Sync()
```

Reply:
yes

Reply:
Using `_ = log.Sync()` might be better to me.